### PR TITLE
feat: allow serialize blessed objects to JSON

### PR DESCRIPTION
**Pull request by beccon4**
prepares for using blessed objects as parameters - proposal to solve issue #58

### CODE REVIEW

File: lib/Raisin/Encoder/JSON.pm [hunk 1]
```diff
@@ -3,12 +3,16 @@ package Raisin::Encoder::JSON;
 use strict;
 use warnings;
 
-use JSON qw(encode_json decode_json);
+use JSON::MaybeXS  qw(encode_json decode_json );
```
Comment (khrt) [inline lib/Raisin/Encoder/JSON.pm]: The code isn't using `encode_json` and `decode_json` functions any longer hence you could safely remove it.

File: lib/Raisin/Encoder/JSON.pm [hunk 2]
```diff
@@ -3,12 +3,16 @@ package Raisin::Encoder::JSON;
...
+my $json=JSON::MaybeXS->new();
```
Comment (khrt) [inline lib/Raisin/Encoder/JSON.pm]: And just minor styling issue, could you please put one space around `=`?

Comment (beccon4) [reply lib/Raisin/Encoder/JSON.pm]: done

File: lib/Raisin/Encoder/JSON.pm (hunk 1 above)
Comment (beccon4) [reply lib/Raisin/Encoder/JSON.pm]: done, thx

**Comment by khrt** [comment]
Thanks for the contribution! I'll try to release new version to CPAN with your other PR over the weekend.

**Pull request merged**